src/main/kotlin/org/jetbrains/research/ml/codetracker/reporting/GitHubAccessTokenScrambler.kt

```diff
@@ -0,0 +1,26 @@
+package org.jetbrains.research.ml.codetracker.reporting
+
+import org.apache.commons.codec.binary.Base64
+import java.io.InputStream
+import java.io.ObjectInputStream
+import java.nio.charset.StandardCharsets
+import javax.crypto.Cipher
+import javax.crypto.spec.IvParameterSpec
+import javax.crypto.spec.SecretKeySpec
+
+/**
+ * Provides functionality to decode secret token.
+ */
+internal object GitHubAccessTokenScrambler {
+    private const val initVector = "RandomInitVector"
+    private const val key = "GitHubErrorToken"
+
+    @Throws(Exception::class)
```

Comment:
Is it possible to clarify what exact exception can be thrown?
Also, I think it's better to use 'throws' annotations everywhere(so mark the rest of our code with it) or nowhere (so remove this annotation), because it's more about code style which should be the same throughout the project.

Reply:
The code can throw a decoding error or you can get an error if the access token incorrect. But I will delete it according to our code style project, thanks!